python_modules/libraries/dagster-shared/dagster_shared/serdes/objects/definition_metadata.py

```diff
@@ -15,6 +16,8 @@ class DgAssetMetadata:
     group: Optional[str]
     description: Optional[str]
     automation_condition: Optional[str]
+    tags: Sequence[tuple[str, str]]
+    metadata: Sequence[tuple[str, str]]
```

Comment:
these don't get persisted or go across IPC boundaries now right? If they did you would need a default value here or they wouldn't deserialize without an error

I think they may not even need whitelist_for_serdes anymore now?